src/Main.elm

```diff
@@ -44,13 +44,20 @@ main =
 type alias Model =
     { key : Browser.Navigation.Key
     , page : Page
+    , viewportWidth : Float
     , emergencyPopupIsOpen : Bool
     }
 
 
 init : () -> Url.Url -> Browser.Navigation.Key -> ( Model, Cmd Msg )
 init _ url key =
-    ( { key = key, page = pageFromMaybeRoute (Route.fromUrl url), emergencyPopupIsOpen = False }, Cmd.none )
+    ( { key = key
+      , page = pageFromMaybeRoute (Route.fromUrl url)
+      , viewportWidth = 500
```

Comment:
This is the temporary part

Reply:
It could become `Device = IsMobile | IsTablet` instead of a Float - if more useful and we could pass that around instead of the viewport width. 

But I'll just get it calculating the width and dumping in there for now.

Reply:
Will this need some JS to get the width from the browser? If the GetHelpPage is the only one that really needs it, conditional display with CSS might be easier

Reply:
Elm has built in viewport functions to get from window and this will be easier to test. It's more usual to do this conditional display than media queries in Elm.

Reply:
The emergency box and the get help page both care, but only if it is mobile - so I was also considering a boolean 'isMobileViewport` rather than the float, but dumping the float is first step anyway?

Reply:
I do think it's a lot easier to just use a little CSS in the two places than pulling a value through the model, but if you want to test for it I guess it needs to know.

Reply:
if this is just for the tel: link, `isMobile` makes sense because we only care if the thing we're on should make a phone call

Reply:
Do you want me to change `viewportWidth` to `isMobile` ?